apps/easy-messe/components/OfferMass/DateTimeMass/TimeClock.tsx

```diff
@@ -23,6 +26,7 @@ function Time({
             timeStep={15}
             onChange={(newTime) => handleTime(newTime)}
             shouldDisableTime={(newTime) => !isTimeAllowed(newTime.toDate()) as boolean}
+            ampm={activeLanguage === 'fr' ? false : true}
```

Comment:
the condition already returns true or false.

```suggestion
            ampm={activeLanguage === 'fr'}
```